app/indexers/work_indexer.rb

```diff
@@ -104,17 +104,17 @@ def inherit_fields solr_doc
       parent_doc = SolrDocument.find(parent_work.id)
       # Loop through all inheritable fields
       ScoobySnacks::METADATA_SCHEMA.inheritable_fields.each do |field|
-        
         #if child value is present and ADD inheritance holds good, add parent value to the child value
-        if (solr_doc[field.solr_name].present? && ScoobySnacks::METADATA_SCHEMA.add_parent_value_display_field_names.include?(field.name))
+        if (solr_doc[field.solr_name]!= parent_doc[field.solr_name] && ScoobySnacks::METADATA_SCHEMA.add_parent_value_display_field_names.include?(field.name))
           solr_doc[field.solr_name].push(*parent_doc[field.solr_name])
-          
-        #if child value is present and no ADD inheritance, skip
-        elsif solr_doc[field.solr_name].present?
-          next
-        #if no child value present, inherit from parent
-        else
-          solr_doc[field.solr_name] = parent_doc[field.solr_name]
```

Comment:
Don't we still need this for non-ADD inheritance?

Reply:
Not sure what has changed, but hyrax is using "inherit_metadata" in app/models/work.rb for non-ADD inheritance. The solr document is updated with child values in work_indexer.rb already.

Reply:
Also, this comment mentions the same. https://github.com/UCSCLibrary/ucsc-library-digital-collections/blob/master/app/indexers/work_indexer.rb#L93

Reply:
I see this comment:
```
  # Metadata inheritance can also happen when the work is saved, in which case the
  # inherited metadata is stored in Fedora and indexed normally.
  # This method comes in to play when you want the inherited metadata to display for users
  # but not to be part of the preservation record.
```
which explains why its in two places. 

So I guess non-add inheritance is saved to the fedora record, and add inheritance just to the solr index (ie searchable and on the website). I wonder if that is intentional? @rmjaffe ?

Reply:
Eeps, I have no idea.  Is there be any need or advantage to treating the add inheritance differently than the other inherited values?

Reply:
ADD-inheritance did not exist previously. We can save that to fedora record if we need to. 
I can explain this over a call on Tuesday.

Reply:
Cool let me know what you come up with - consider this PR approved pending this decision.